agrirouter-api-java-convenience/src/main/java/com/dke/data/agrirouter/convenience/mqtt/client/MqttOptionService.java

```diff
@@ -62,7 +62,8 @@ public MqttConnectOptions createMqttConnectOptions(OnboardingResponse onboarding
   private SocketFactory getSocketFactory(
       String certificate, String password, CertificationType certificationType) {
     try {
-      KeyManagerFactory kmf = KeyManagerFactory.getInstance(KeyManagerFactory.getDefaultAlgorithm());
+      KeyManagerFactory kmf =
```

Comment:
Fomatter issues?

Reply:
Should have been corrected by mvn clean install, however, fixed it by hand.

Reply:
It's formatted like that if I do mvn clean install, so, the "problem" is at least part of the project